app/components/card/text-box.gjs

```diff
@@ -1,5 +1,7 @@
 import { htmlSafe } from '@ember/template';
+import { LinkTo } from '@ember/routing';
 import FaIcon from '@fortawesome/ember-fontawesome/components/fa-icon';
+import { hash } from 'rsvp';
```

Comment:
Ember templates don't have syntax (as of yet) for declaring an inline object ("hash" as in hashmap), so the helper you need to import is from this list: https://github.com/ember-template-imports/ember-template-imports#reference-built-in-helpers-modifiers-components

```suggestion
import { hash } from '@ember/helper';
```